Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityArticulatedHand.cs

```diff
@@ -86,17 +97,16 @@ public override bool IsInPointingPose
                         valid = false;
                     }
                 }
-#endif // WINDOWS_UWP
+#endif // (UNITY_WSA && DOTNETWINRT_PRESENT) || WINDOWS_UWP
                 return valid;
             }
         }
 
 #if UNITY_WSA
+#if DOTNETWINRT_PRESENT || WINDOWS_UWP
```

Comment:
nit: previous conditionals have DOTNETWINRT_PRESENT second. not a required change, though it may help with readability

Reply:
Updated to `WINDOWS_UWP || DOTNETWINRT_PRESENT`